doc/release_instructions.md

```diff
@@ -52,4 +54,19 @@ make point-release
 
 Afterwards, you'll be prompted to push your changes to master. Please do so.
 
+## Custom Release
+
+Occasionally we have a need for a custom release off a non master branch. This is usually because some features we need to supply to users that are incompatible w features on master, but the features on master we need to keep longer term.
+
+Follow the instructions above, just run the make script off of your branch. 
+
+Then after a successful release:
+
+- The tag created by goxc will point to master, and this is wrong. Opened an issue for this: https://github.com/laher/goxc/issues/112
+  - To mitigate this, you'll have to go to github.com/pachyderm/pachyderm/releases, and manually delete the release that goxc created
```

Comment:
add `https://` to `github.com/pachyderm/pachyderm/releases` so autolinking works